app/shared/components/register.js

```diff
@@ -80,6 +80,28 @@ function registerPlaysOverview({ plays }) {
     );
 }
 
+function imageComponent(imageList, date){
+    const hasImg = imageList.length > 0;
+    let imageComp = null;
+    let pagenum = null;
+    let rnum = null;
+
+    if(hasImg){
+        pagenum = /.*?_([0-9]+)\.jpg/.exec(imageList[0].url)[1];
+        rnum = imageList[0].rnum;
+
+        imageComp = ce('a', {
+            href: `http://hyperstudio.mit.edu/cfrp/flip_books/${rnum}/#page/${pagenum}/mode/1up`,
+            title: 'Consulter la page du registre pour la soirée du '+(date),
+            target: "_blank"
+        },
```

Comment:
we use an eslint config that requires trailing commas in es6 code and indentation looks strange. Could you run `eslint` / `prettier-eslint` on the code and fixes corresponding warnings / errors ?